test/unit/gateways/mundipagg_test.rb

```diff
@@ -67,6 +93,17 @@ def test_successful_purchase_with_holder_document
     assert response.test?
   end
 
+  def test_successful_purchase_with_submerchant
+    options = @options.update(@submerchant_options)
+    response = stub_comms do
+      @gateway.purchase(@amount, @credit_card, options)
+    end.check_request do |_endpoint, data, _headers|
```

Comment:
Hey is this supposed to have something in it?

Reply:
Radically revised!